### PR TITLE
Filled incomplete sentence.

### PR DESCRIPTION
**Pull request by xBorderie**
ping @AlexEven just in case.

### CODE REVIEW

**Comment by maximebiloe** [comment]
Not the right branch @xBorderie 😉

File: ps_imageslider.php [hunk 1]
```diff
@@ -858,7 +858,7 @@ public function renderForm()
                         'type' => 'switch',
                         'label' => $this->getTranslator()->trans('Pause on hover', array(), 'Modules.ImageSlider'),
                         'name' => 'HOMESLIDER_PAUSE_ON_HOVER',
-                        'desc' => $this->getTranslator()->trans('Stop sliding when .', array(), 'Modules.ImageSlider'),
+                        'desc' => $this->getTranslator()->trans('Stop sliding when  when the mouse cursor is over the slideshow.', array(), 'Modules.ImageSlider'),
```
Comment (maximebiloe) [inline ps_imageslider.php]: double "when"? ^^

**Comment by xBorderie** [comment]
Double damn! Thanks @maximebiloe and @julienbourdeau :)

**Comment by AlexEven** [comment]
Could we please merge this one?  :innocent:
I'd like to update Crowdin content this week. Thanks!

**Pull request merged**